motr/io_nw_xfer.c

```diff
@@ -513,6 +517,8 @@ static void target_ioreq_seg_add(struct target_ioreq              *ti,
 	toff    = target_offset(frame, play, gob_offset);
 	pgstart = toff;
 	goff    = unit_type == M0_PUT_DATA ? gob_offset : 0;
+	coff    = di_cksum_offset(play, gob_offset);
```

Comment:
I think we can move this computation inside the while loop as goff changes in while loop
coff    = di_cksum_offset(play, goff);